<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EchoWindow</class>
 <widget class="QMainWindow" name="EchoWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>222</width>
    <height>308</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QFormLayout" name="formLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="rateLbl">
        <property name="text">
         <string>Sample rate</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="rate">
        <property name="currentText">
         <string>8 000</string>
        </property>
        <item>
         <property name="text">
          <string>8 000</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>11 025</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>16 000</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>22 050</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>32 000</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>44 100</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>48 000</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>96 000</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>192 000</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>2 822 400</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>5 644 800</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="sizeLbl">
        <property name="text">
         <string>Sample size</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QComboBox" name="size">
        <property name="currentText">
         <string>8bit</string>
        </property>
        <item>
         <property name="text">
          <string>8bit</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>16bit</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>32bit</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QComboBox" name="type">
        <item>
         <property name="text">
          <string>SignedInt</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>UnSignedInt</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Float</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="typeLbl">
        <property name="text">
         <string>Sample type</string>
        </property>
       </widget>
      </item>
      <item row="7" column="0">
       <widget class="QLabel" name="chOutLbl">
        <property name="text">
         <string>Channels output</string>
        </property>
       </widget>
      </item>
      <item row="7" column="1">
       <widget class="QSpinBox" name="chOut">
        <property name="minimum">
         <number>1</number>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="inputLbl">
        <property name="text">
         <string>Input</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QComboBox" name="devIn"/>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="OutputLbl">
        <property name="text">
         <string>Output</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QComboBox" name="devOut"/>
      </item>
      <item row="6" column="0">
       <widget class="QLabel" name="chInLbl">
        <property name="text">
         <string>Channels input</string>
        </property>
       </widget>
      </item>
      <item row="6" column="1">
       <widget class="QSpinBox" name="chIn">
        <property name="minimum">
         <number>1</number>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QPushButton" name="onOffBtn">
      <property name="text">
       <string>Start echo.</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>222</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
